django_project/frontend/src/containers/MainPage/SideBar/LayerFilterTabs.tsx

```diff
@@ -17,12 +17,14 @@ function LayerFilterTabs(props: { selectedMainTabIdx: number }) {
 
         // Check if the URL contains a pattern that indicates the presence of charts
         if (
-            currentUrl.includes('/charts') || 
-            currentUrl.includes('/?tab=2')
+            currentUrl.includes('/charts') ||
+            currentUrl.includes('/reports') ||
+            currentUrl.includes('/?tab=2') ||
+            currentUrl.includes('/?tab=1')
```

Comment:
Indeed, but it will need more time as I'm trying to understand the URL conversion here, where it happened etc.
This occured when I refresh reports and charts. First the URL is changed to /map?tab=1, then to /reports. Same thing with charts, /map?tab=2, then to /reports. So, technically, every URL has map keyword in it.
[Kazam_screencast_00029.webm](https://github.com/kartoza/sawps/assets/7352963/15e44a7c-b47a-4e91-b5a8-ccaed295df4e)

Reply:
Yes, I think the layer panel will show up and then disappear. It should be fine for now. Can you change it to currentUrl.includes('/map')?

Reply:
I can't. I use this and it works
`currentUrl.endsWith('/map') ||
          currentUrl.endsWith('/map/')`